### PR TITLE
update documents

### PR DESCRIPTION
**Pull request by hvthhien**

### CODE REVIEW

File: docs/Feature.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Using GordianSigner
+
+*GordianSigner* allows you to: sign a PSBT, import & managing your accounts, fill & export account map and save contacts.
+This instructions show how to use these basic features.
+
+## Singing a PSBT
+
+<img src="images/sign_psbt.png" alt="" width="250"/> <img src="images/sign_psbt_import.png" alt="" width="250"/>
+
+You can import a PSBT to sign by pasting from clipboard, import from file or scan UR code.
```
Comment (jollyjoker992) [inline docs/Feature.md]: It should be `importing` and `scanning` as well.

Comment (hvthhien) [reply docs/Feature.md]: There are lots of ways to import a PSBT here, so I think just showing the result after importing/scanning & describe in the description is enough. Anyway, these steps are very common like scanning or browsing file.

Comment (jollyjoker992) [reply docs/Feature.md]: @hvthhien I mean the spelling mistake.

Comment (hvthhien) [reply docs/Feature.md]: okay, I see

**Review by jollyjoker992 — APPROVED** [review]

**Pull request merged**